### ISSUE TITLE
is suport sdxl?

### ISSUE THREAD

**Issue by libai-lab** [fallenshock/FlowEdit#3]
is suport sdxl?

**Comment by Jiahao620** [comment]
Similar question. Does FlowEdit support stable diffusion 2.1?

**Comment by zhang2002forwin** [comment]
> FlowEdit is currently implemented only for flow models and not diffusion models. Although FlowEdit can be also implemented for diffusion models, there are some differences in the formulation between diffusion and flow processes, hence it doesn't suppose to work out-of-the-box and there is no current support to SDXL, SD2.1, etc.

Excuse me, if we modify the FlowEdit editing process according to the iterative formula of the diffusion model, can we get a good editing effect?

**Comment by matankleiner** [comment]
Yes, but one should do it carefully, due to the differences between the $$V$$ and $$\epsilon$$ prediction. 
Mainly, $$V_t^{\Delta}$$ in terms of epsilon prediction is 

$$
V_t^{\Delta}(\hat{Z}_t^{\text{src}}, \hat{Z}_t^{\text{tar}}) = \frac{1}{1-t} \left( \left(\hat{\epsilon}(\hat{Z}_t^{\text{tar}}, t) - \hat{\epsilon}(\hat{Z}_t^{\text{src}}, t)\right) - (Z_t^{\text{FE}} - X_0^{\text{src}} ) \right)
$$

**Comment by zhang2002forwin** [comment]
> Yes, but one should do it carefully, due to the differences between the 
> 𝑉
>  and 
> 𝜖
>  prediction.
> Mainly, 
> 𝑉
> 𝑡
> Δ
>  in terms of epsilon prediction is

Thans a lot!!   This formula satisfies the epsilon prediction, right?

**Comment by matankleiner** [comment]
Yes
